src/knex/filters.js

```diff
@@ -55,55 +56,55 @@ const negate = definitions => {
 }
 
 const eq = (ctx, queryKey, dbKey = queryKey, transform = v => v) => {
-  if (typeof ctx.query[queryKey] !== 'undefined') {
-    const transformedValue = transform(convertStringToNull(ctx.query[queryKey]))
-
-    return knexBuilder =>
-      transformedValue === null
-        ? knexBuilder.whereNull(dbKey)
-        : knexBuilder.where(dbKey, transformedValue)
+  if (typeof ctx.query[queryKey] === 'undefined') {
```

Comment:
why not check for `null` as well?

Reply:
it's impossible for a query param to be `null` there, it will be a string/array or undefined